src/main/kotlin/com/makarytskyi/rentcar/exception/GlobalExceptionHandler.kt

```diff
@@ -0,0 +1,33 @@
+package com.makarytskyi.rentcar.exception
+
+import java.time.LocalDateTime
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+
+@RestControllerAdvice
+internal class GlobalExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException::class)
+    internal fun handleResourceNotFoundException(): ResponseEntity<ErrorResponse> {
+        val errorResponse = ErrorResponse(
+            LocalDateTime.now(),
```

Comment:
why this timestamp is needed?

Reply:
to get time when exception happened

Reply:
do you use it somehow? if not please clean it out